packages/components/src/Menu/Menu.tsx

```diff
@@ -336,30 +356,45 @@ function SectionHeader({
   );
 }
 
-function Action({
-  label,
-  sectionLabel,
-  icon,
-  iconColor,
-  destructive,
-  UNSAFE_style,
-  UNSAFE_className,
-  onClick,
-}: ActionProps) {
-  const actionButtonRef = useRef() as RefObject<HTMLButtonElement>;
```

Comment:
this ref was a leftover from when we used to use a `useEffect` to focus the first item

we haven't used these in a very long time. I just forgot to remove them when I made the changes back then.